tests/assets/db/invalid/semantics/missing_case/suite.yaml

```diff
@@ -1,6 +1,3 @@
 description: suite1
 cases:
     - name: case1
-patterns:
-    - pattern: a
-      case_name: case2
```

Comment:
We need to remove this database and the associated test case altogether as they no longer apply. The test should have a TODO about this.